package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "bluebird": "^3.3.1",
     "commander": "^2.9.0",
-    "cucumber": "^1.0.0",
+    "cucumber": "1.0.0",
```

Comment:
We should, and we should bump the major version of this repo because mutil-cuke allows you to specify the cucumber you want to use and the newer version of cucumber creates a backwards incompatible output.